pkg/build/builder/daemonless.go

```diff
@@ -67,6 +146,9 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 		BlobDirectory: blobCacheDirectory,
 	}
 	_, err = buildah.Pull(context.TODO(), "docker://"+imageName, options)
+	if err := os.Remove(credsFile); err != nil {
```

Comment:
this should be in a defer immediately after the err check following `mergeNodeCredentials`

Reply:
Right, fixed.

Reply:
looks good